endpointmanager/pkg/versionsoperatorparser/versionsoperatorparser_test.go

```diff
@@ -0,0 +1,91 @@
+package versionsoperatorparser
+
+import (
+	"testing"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/helpers"
+	th "github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/testhelper"
+)
+
+func Test_Equal(t *testing.T) {
+	var vr1 VersionsResponse
+	var vr2 VersionsResponse
+
+	var equal bool
+
+	vr1 = VersionsResponse{Response: nil}
+	vr2 = VersionsResponse{Response: nil}
+
+	// test both nil equal
+	equal = vr1.Equal(vr2)
+	th.Assert(t, equal, "expected equality nil to nil to be true")
+
+	// test one nil, other not
+	var resp map[string]interface{} = make(map[string]interface{})
+	resp["versions"] = "[\"4.0\"]"
+	resp["default"] = "4.0"
+	vr2 = VersionsResponse{Response: resp}
+	equal = vr1.Equal(vr2)
+	th.Assert(t, !equal, "expected equality nil to not nil to be false")
+
+	// test two same non-nil
+	var resp1 map[string]interface{} = make(map[string]interface{})
+	resp1["versions"] = "[\"4.0\"]"
+	resp1["default"] = "4.0"
+	vr1 = VersionsResponse{Response: resp1}
+	equal = vr1.Equal(vr2)
+	th.Assert(t, equal, "expected equality for same Response to be true")
+
+	// test two different non-nil
+	resp1["versions"] = "[\"1.0\"]"
+	resp1["default"] = "1.0"
+	vr1 = VersionsResponse{Response: resp1}
+	equal = vr1.Equal(vr2)
+	th.Assert(t, !equal, "expected equality for different Response to be false")
+}
+
+func Test_GetDefaultVersion(t *testing.T) {
+	var vr1 VersionsResponse
+
+	var equal bool
+
+	var resp map[string]interface{} = make(map[string]interface{})
+	resp["versions"] = "[\"1.0\"]"
+	resp["default"] = "4.0"
+	vr1 = VersionsResponse{Response: resp}
+
+	// test populated versions Response
+	equal = vr1.GetDefaultVersion() == "4.0"
+	th.Assert(t, equal, "expected default version to be 4.0")
+
+	vr1 = VersionsResponse{Response: nil}
```

Comment:
This test looks the same as the empty test for GetSupportedVersions, I think this test should be checking for an empty string for GetDefaultVersion

Reply:
good catch